site/routes/index.js

```diff
@@ -32,13 +32,19 @@ function prefixRoutes(rs) {
 }
 
 export default function routes({ contactUsURL }) {
+  const titleSuffix = process.env.COMPANY || 'Red Badger';
```

Comment:
Not sure `COMPANY` really needs to be an environment variable, particularly as we have a fall back here. @asavin ?